src/node_options.cc

```diff
@@ -789,6 +789,13 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "Experimental type-stripping for TypeScript files.",
             &EnvironmentOptions::experimental_strip_types,
             kAllowedInEnvvar);
+  AddOption("--experimental-transform-types",
+            "enable transformation of TypeScript-only"
+            "syntax in JavaScript code",
```

Comment:
Is this already good or would
```suggestion
            "syntax to JavaScript code",
```
technically be more correct?